<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string>
    QWidget {
        background-color: #F2F2F2;
    }
    QLabel#video_label {
        background-color: #243E73;
        color: white;
        border-radius: 8px;
    }
    QPushButton {
        background-color: #049DD9;
        color: #F2F2F2;
        border-radius: 8px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #8BBBD9;
    }
    QPushButton:pressed {
        background-color: #243E73;
    }
   </string>
  </property>
  <widget class="QLabel" name="video_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>1071</width>
     <height>581</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ba≈ülat/Durdur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tekrar Oynat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>620</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
